src/crafting_gui.cpp

```diff
@@ -450,6 +462,13 @@ static std::string practice_recipe_description( const recipe &recp,
     return oss.str();
 }
 
+static std::string nested_recipe_description( const recipe &recp )
+{
+    std::ostringstream oss;
+    oss << recp.description.translated() << "\n\n";
```

Comment:
You don't need a string stream here, you can just concatenate the strings:
```suggestion
    return recp.description.translated() + "\n\n";
```
And for that you don't even need a function, you can use this line directly.